paasta_tools/cli/cmds/local_run.py

```diff
@@ -20,6 +20,7 @@
 import time
 import uuid
 from os import execlp
+from pathlib import Path
```

Comment:
Would [os.path.expanduser](https://docs.python.org/3/library/os.path.html#os.path.expanduser) not be sufficient? i.e. `os.path.expanduser('~/.dockercfg')`

Reply:
Yea, looks like the same. I'll use that method to save an import